### PR TITLE
React/pt 68/oauth last part

### PR DESCRIPTION
**Pull request by gbasile**
TL;DR;

Completed Oauth Flow
Create an AuthenticationService that provide information about the logged user
Update Debug Screen: Every time we authenticate now we store the information into the AuthenticationService so that they are persisted in the app

|iOS|Android|
|---|---|
|Part 1| Part 1|
|1st time ios | 0 time android |
|Part 2| Part 2|
| 2nd time ios | 1st time android |

### CODE REVIEW

File: ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js [hunk 1]
```diff
@@ -3,6 +3,28 @@ jest.unmock('../../../core/service/percent-encoder.js')
 // var OauthHelper = require('../../../core/service/oauth-helper.js')
 
 describe('OauthHelper', () => {
+  // describe('URL Callback', () => {
```
Comment (gbasile) [inline ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js]: again, we write some tests but we have to comment them

**Comment by danybony** [comment]
Nice work! 👍

File: ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js (hunk 1 above)
Comment (frapontillo) [inline ReactNative/ReactTwitter/__tests__/core/service/oauth-helper.test.js]: Have you opened a bug on the react-native repo regarding the impossibility to test modules that require react-native components?

**Comment by frapontillo** [comment]
LGTM 👍 leaving the merge to someone in the react-native team.

**Pull request merged**